docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-alpine3.8
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+ENV AWS_CONFIG_FILE=/root/.aws/config
+ENV NEO4J_HOST=neo4j
+
+ADD https://raw.githubusercontent.com/eficode/wait-for/master/wait-for .
+
+CMD sh wait-for ${NEO4J_HOST}:7474 -- cartography --neo4j-uri bolt://${NEO4J_HOST}:7687 --aws-sync-all-profiles
```

Comment:
I think `--aws-sync-all-profiles` should be removed from here and we can make it clear in the docs that the Dockerfile is only meant as a quick playground so users don't need to install neo4j.

As covered [here](https://github.com/lyft/cartography#multiple-aws-account-setup), multiple AWS account setup can be very complicated and I'd rather not cover it in the docker setup.

Reply:
Sure, I just added a commit removing this flag.